### PR TITLE
Run Linter Changes

### PR DESCRIPTION
**Pull request by skgbafa**
# Description

This PR has runs the linter changes made in #26. There are no functionality changes in this PR.

# Type

- [x] Chore (non-breaking change which improves the codebase)

# Diligence Checklist
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings

### CODE REVIEW

File: packages/ssx-core/src/client/index.ts [hunk 1]
```diff
@@ -1 +1 @@
-export * from './types';
\ No newline at end of file
+export * from './types';
```
Comment (w4ll3) [inline packages/ssx-core/src/client/index.ts:1]: This goes against POSIX, lines must end in a new line.

File: packages/ssx-core/src/index.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
 export * from './types';
 export * from './utils';
 export * from './client';
-export * from './server';
```
Comment (w4ll3) [inline packages/ssx-core/src/index.ts:4]: Lines must end in a new line

File: packages/ssx-core/src/server/types.ts [hunk 3]
```diff
@@ -88,4 +87,4 @@ export enum SSXEventLogTypes {
   // LOG = "LOG",
   /** Event type definition. */
   // EVENT = "event",
-}
```
Comment (w4ll3) [inline packages/ssx-core/src/server/types.ts:91]: Lines must end in a new line

File: packages/ssx-core/src/server/utils.ts [hunk 4]
```diff
@@ -10,33 +10,31 @@ import { SSXLogFields } from "./types";
...
-    data: SSXLogFields
+  api: AxiosInstance,
+  apiKey: string,
+  data: SSXLogFields
 ): Promise<boolean> => {
-    if (!data.timestamp) data.timestamp = new Date().toISOString();
-    return (
-        Boolean(apiKey) && ssxPost(api, '/events', data)
-    );
-};
```
Comment (w4ll3) [inline packages/ssx-core/src/server/utils.ts:42]: Lines must end in a new line

File: packages/ssx-core/src/types.ts [hunk 5]
```diff
@@ -187,7 +201,7 @@ export interface SSXEnsResolveOptions {
 /** ENS data supported by SSX. */
 export interface SSXEnsData {
   /** ENS name/domain. */
-  domain?: string | null,
+  domain?: string | null;
   /** ENS avatar. */
-  avatarUrl?: string | null
-}
```
Comment (w4ll3) [inline packages/ssx-core/src/types.ts:193]: Lines must end in a new line

**Review by w4ll3 — CHANGES_REQUESTED** [review]
Looks good, just must fix the files to end with an empty line to follow POSIX standards.

File: packages/ssx-core/src/client/index.ts (hunk 1 above)
Comment (samgbafa) [reply packages/ssx-core/src/client/index.ts:1]: Checked on this and they do! Github, just renders them as a single line, if they don't have a newline the red minus, like this did

Screenshot 2022-12-07 at 2 30 00 PM

Comment (samgbafa) [reply packages/ssx-core/src/client/index.ts:1]: Will resolve mentions of this issue!

**Review by Juliano1612 — APPROVED** [review]

**Review by w4ll3 — APPROVED** [review]

**Pull request merged**